backend/src/v5/processors/teamspaces/index.js

```diff
@@ -43,9 +43,11 @@ const { removeUserFromAllProjects } = require('../../models/projectSettings');
 const Teamspaces = {};
 
 const removeAllUsersFromTS = async (teamspace) => {
-	const members = await getMembersInfo(teamspace);
+	const { refId: tenantId } = await getTeamspaceSetting(teamspace, { refId: 1 });
```

Comment:
I think what you really want here is a common function that getAllMembersInTeamspace

Which will do the frontegg stuff and convert all the userIds to usernames.

Then things like this function, `getTeamspaceMembersInfo` will just call that common function.

There's a lot of duplication everywhere just to get the members list.